libs/category/src/filters/find-applied/find-applied-options.ts

```diff
@@ -0,0 +1,10 @@
+import {
+  DaffCategoryFilter,
+  DaffCategoryFilterOption,
+} from '../../models/public_api';
+
+/**
+ * Finds the applied options in the filter.
+ */
+export const daffCategoryFindAppliedFilterOptions = (filter: DaffCategoryFilter): DaffCategoryFilterOption[] =>
```

Comment:
I think this should be using `DaffCategoryFilterOptionBase` instead.

Reply:
Take a look at the new type. This is fixed.